pkg/backup/backup_compaction_test.go

```diff
@@ -105,11 +147,19 @@ func TestBackupCompaction(t *testing.T) {
 			db.Exec(t, "DROP TABLE foo")
 		}()
 		db.Exec(t, "INSERT INTO foo VALUES (1, 1), (2, 2), (3, 3)")
-		db.Exec(t, fullBackupStmt)
+		start := getTime()
+		db.Exec(t, fmt.Sprintf("BACKUP INTO 'nodelocal://1/backup' AS OF SYSTEM TIME '%s'", start))
+
```

Comment:
could you add an index here that you subsequently drop, captured in the next backup?